src/topology/uniform_space/equicontinuity.lean

```diff
@@ -0,0 +1,341 @@
+/-
+Copyright (c) 2022 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anatole Dedecker
+-/
+import topology.uniform_space.uniform_convergence_topology
+
+/-!
+# Equicontinuity of a family of functions
+
```

Comment:
Can you recall in the docstring what you mean by equicontinuity? Ideally, the docstring should be readable by someone who doesn't know the notion a priori.